app/src/main/java/de/westnordost/streetcomplete/screens/settings/questselection/QuestSelectionAdapter.kt

```diff
@@ -80,15 +89,22 @@ class QuestSelectionAdapter(
             }
         }
 
+    private fun questTypeMatchesSearchWords(questType: QuestType, words: List<String>): Boolean {
+        val question = genericQuestTitle(context.resources, questType).lowercase()
+        if (question.containsAll(words)) {
+            return true
+        }
+
+        val englishQuestion = genericQuestTitle(englishResources, questType).lowercase()
+        return englishQuestion.containsAll(words)
+    }
```

Comment:
```suggestion
    private fun questTypeMatchesSearchWords(questType: QuestType, words: List<String>) =
        genericQuestTitle(context.resources, questType).lowercase().containsAll(words)
        || genericQuestTitle(englishResources, questType).lowercase().containsAll(words)
```

I think this is well readable, too

Reply:
Yes, that's actually much nicer than my version!